lambdas/sessionstart/src/main/java/uk/gov/di/ipv/core/session/IpvSessionStartHandler.java

```diff
@@ -82,6 +90,8 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             String ipvSessionId = ipvSessionService.generateIpvSession(clientSessionDetails);
 
+            auditService.sendAuditEvent(AuditEventTypes.IPV_JOURNEY_START);
```

Comment:
You'll also need to give the SessionStart lambda permission to send messages to SQS.

Example of that here: https://github.com/alphagov/di-ipv-core-back/blob/main/deploy/template.yaml#L326-L327